setup.py

```diff
@@ -1,11 +1,28 @@
 # setup.py
 from setuptools import Extension, find_packages, setup
+import sys
+
+extra_compile_args=[]
+libraries=[]
+
+# call python setup.py -coverage install to install with coverage enabled.
+# and debug symbols; coverage info will be generated in
+# bcachefs/build/temp.linux-x86_64-3.8/bcachefs/*.(gcda|gcno)
+if '-coverage' in sys.argv:
+    print("Compiling with coverage")
+    sys.argv.remove('-coverage')
+
+    extra_compile_args=["-coverage", "-g3", "-O0"]
+    libraries=["gcov"]
+
```

Comment:
Minor but this is an extra new line

Reply:
I like to put an extra line after a if :)